index.php

```diff
@@ -1,13 +1,49 @@
 <?php
-require 'functions.php';
+include_once 'functions.php';
+?>
 
-echo '<h1>Record Collection</h1>';
-echo '<br>';
-echo '<br>';
-
-$db = getDB();
-$query = $db->prepare('SELECT `Title`, `Rating`, `Released`, `Condition`, `Artist` FROM `records`;');
-$query->execute();
-$records = $query->fetchALL();
-
-displayRecords($records);
\ No newline at end of file
+<html lang="en">
+<!DOCTYPE html>
+<html>
```

Comment:
you have two html tags, this one and the one on line 5, you should only have one